docs/includes/manifest-guidance.md

```diff
@@ -0,0 +1,7 @@
+> [!TIP]
+> If you'll be testing your add-in across multiple environments (for example, in development, staging, demo, etc.), we recommend that you maintain a different XML manifest file for each environment. In each manifest file, you can:
+> - Specify the URLs that correspond to the environment.
+> - Customize metadata values like `DisplayName` and labels within `Resources` to indicate the environment (so that end users will be able to identify the environment a sideloaded add-in corresponds to). 
```

Comment:
Don't know if you need the ()s here? Also getting rid of preposition at end of sentence
```suggestion
> - Customize metadata values like `DisplayName` and labels within `Resources` to indicate the environment, so that end users will be able to identify a sideloaded add-in's corresponding environment. 
```